### ISSUE TITLE
USF glider data QC concern

### ISSUE THREAD

**Issue by robragsdale** [ioos/ioosngdac#92]
Hi Rob-
I noticed with some concern that glider data going from USF into the database is not really QC'd. Specifically for example, salinity is not temperature corrected - the CTD data is sent uncorrected.

Chad mentioned this is something everyone is doinng (i.e. the CTD data is not corrected).

This is not good. After a few years NOBODY will be able to process this quantity of data, and worse, some people are likely already using this data for analyses (climate change etc would be distorted, and other bad analyses will propagate)

Can you address this issue in your group?

thanks
Frank

Chad Lembke wrote:
Frank,
This just came out earlier this month, shortly after our discussions.
We'll be looking into it to see how we can best adapt.
Any suggestions are welcome.

Rob,
feel free to add on as needed.

Thanks,
Chad

Chad-
someone simply has to do the work to match the temperature and salinity, like we all do when you collect CTD casts in oceanography. This is not different
Frank

**Comment by kerfoot** [comment]
Not sure who Frank is, but there are few issues with his statements:
1. It's an incorrect statement that nobody is QCing their data.
2. All of our discussions have come to the conclusion that the responsibility for QC is on the data  provider, not the DAC.  If we QC someone elses data, we will get yelled at.  If we don't QC some elses data, we will get yelled at.  Classic no-win situation.
3. The statement that "someone simply has to do the work to match temperature and salinity, like we all do when you collect CTD casts in oceanography.  This is not different" is naive, at best, and flat out wrong at worst.  If it were this easy, it would have been done a long time ago.  There are a number of reasons why everything about this statement is incorrect, but the 2 biggest ones are the primary errors associated with salinity/density spiking in gliders are lack of vertical resolution and thermal intertia associated with a slow moving CTD across thermal gradients.  It's a complex problem for which I'm not aware of a solution.

**Comment by robragsdale** [comment]
@kerfoot Frank is Frank Muller-Karger from USF.

**Comment by robragsdale** [comment]
Through an email discussion with Frank Muller-Karger, we strived to resolve his apprehension over the release of real-time, uncorrected CTD data.  We explained that there is no easy solution and pointed to current common practices and ongoing work to improve the data quality.  

* The glider DAC team is coordinating with QARTOD to identify quality control measures for temperature/salinity that can be applied.  
* NDBC has checks in place before the data is accepted and allowed onto the GTS; if a glider profile is flagged the data is not released.  
* NCEP is confident using data that is coming from the GTS in their model assimilation.
* We heard from several folks with glider expertise working on the quality control issue that very little correcting of real-time CTD glider data is happening. 

Frank is continuing to discuss this issue with Mark Bushnell who is leading the quality control work in hopes that a solution can be found.
